crates/bevy_core_pipeline/src/taa/mod.rs

```diff
@@ -0,0 +1,534 @@
+use crate::{
+    fullscreen_vertex_shader::fullscreen_shader_vertex_state,
+    prelude::Camera3d,
+    prepass::{DepthPrepass, VelocityPrepass, ViewPrepassTextures},
+};
+use bevy_app::{App, Plugin};
+use bevy_asset::{load_internal_asset, HandleUntyped};
+use bevy_core::FrameCount;
+use bevy_ecs::{
+    prelude::{Bundle, Component, Entity},
+    query::{QueryState, With},
+    schedule::IntoSystemDescriptor,
+    system::{Commands, Query, Res, ResMut, Resource},
+    world::{FromWorld, World},
+};
+use bevy_math::vec2;
+use bevy_reflect::{Reflect, TypeUuid};
+use bevy_render::{
+    camera::{ExtractedCamera, TemporalJitter},
+    prelude::{Camera, Projection},
+    render_graph::{Node, NodeRunError, RenderGraph, RenderGraphContext, SlotInfo, SlotType},
+    render_resource::{
+        BindGroupDescriptor, BindGroupEntry, BindGroupLayout, BindGroupLayoutDescriptor,
+        BindGroupLayoutEntry, BindingResource, BindingType, CachedRenderPipelineId,
+        ColorTargetState, ColorWrites, Extent3d, FilterMode, FragmentState, MultisampleState,
+        Operations, PipelineCache, PrimitiveState, RenderPassColorAttachment, RenderPassDescriptor,
+        RenderPipelineDescriptor, Sampler, SamplerBindingType, SamplerDescriptor, Shader,
+        ShaderStages, SpecializedRenderPipeline, SpecializedRenderPipelines, TextureDescriptor,
+        TextureDimension, TextureFormat, TextureSampleType, TextureUsages, TextureViewDimension,
+    },
+    renderer::{RenderContext, RenderDevice},
+    texture::{BevyDefault, CachedTexture, TextureCache},
+    view::{ExtractedView, Msaa, ViewTarget},
+    Extract, RenderApp, RenderStage,
+};
+#[cfg(feature = "trace")]
+use bevy_utils::tracing::info_span;
+
+mod draw_3d_graph {
+    pub mod node {
+        /// Label for the TAA render node.
+        pub const TAA: &str = "taa";
+    }
+}
+
+const TAA_SHADER_HANDLE: HandleUntyped =
+    HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 656865235226276);
+
+/// Plugin for temporal antialiasing. Disables multisample antialiasing (MSAA).
+pub struct TemporalAntialiasPlugin;
```

Comment:
I think this and every other instance of similarly named things should use `TemporalAntiAlias` It seems weird otherwise that the acronym is TAA.

Reply:
I personally dislike that, but it's an easy change either way.

Reply:
Anti-aliasing counts as 2 words no? It's not a deal breaker or anything, but it feels weird to me if it doesn't map with the acronym. Personally I would just have used Taa though. Just like Fxaa and Msaa aren't spelled out.

Reply:
I feel the same. `Antialias` looks very strange to me compared to `AntiAlias`. And, if it were `TemporalAntialias`, wouldn't the acronym be `TA` instead of `TAA`?